docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "64053c0f-3582-465b-9e4c-a83da332da88",
+   "metadata": {},
+   "source": [
+    "# Find Label Errors in Multilabel (image) Datasets\n",
+    "\n",
+    "This tutorial shows how you can use cleanlab to find potential label errors in multilabel datasets. In such datasets, each instance can be labeled with any of k labels, where k is the total number of different labels in the dataset.  Example applications in NLP include part-of-speech-tagging or entity recognition, which is the focus on this tutorial. Here we use the [CelebA](https://www.kaggle.com/datasets/jessicali9530/celeba-dataset) which consists a large-scale face attributes dataset with more than 200K celebrity images, each with 40 attribute annotations. We use a subset of the attributes in this tuturial \n",
+    "\n",
+    "- Eyeglasses (Person is wearing glasses)\n",
+    "- Wearing_Earrings (Person is wearing earings)\n",
+    "- Wearing_Hat (Person is wearing hat)\n",
+    "- Wearing_Necklace (Person is wearing a necklace)\n",
+    "- Wearing_Necktie (Person is wearing a necktie)\n",
+    "- No_Beard (beard is present)\n",
+    "- Smiling (Person is smiling)\n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "\n",
+    "- Find tokens with label issues using `cleanlab.filter.find_label_issues` with `multi_label = True`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "adaefc8b-b639-4bdf-af0d-337519e37ffc",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "We have `multi_labels` and `pred_probs` from a model trained on an existing data? Run the code below to get improved consensus labels and more information about the quality of your labels and annotators.\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```ipython3 \n",
+    "from cleanlab.filter import find_label_issues\n",
+    "\n",
+    "noise_idx = find_label_issues(\n",
+    "        labels=dataset[\"labels\"],\n",
+    "        pred_probs=dataset[\"pred_probs\"],\n",
+    "        multi_label=True,\n",
+    "        return_indices_ranked_by='normalized_margin',\n",
```

Comment:
How did you decide to use this option vs other options?

Reply:
I tried other return_indices_ranked_by methods, this seemed to have more errors within top 10 issues (indices)